app/controllers/admin/embeddable_hosts_controller.rb

```diff
@@ -28,6 +28,22 @@ def save_host(host, action)
     host.category_id = params[:embeddable_host][:category_id]
     host.category_id = SiteSetting.uncategorized_category_id if host.category_id.blank?
 
+    # Handle tags
+    username = params[:embeddable_host][:user]
+    tags = params[:embeddable_host][:tags]&.map(&:strip)
+
+    if tags.present?
+      host.tags = Tag.where(name: tags)
```

Comment:
This is not working when you're creating a new embeddable host.

`validates :embeddable_host_id, presence: true` is triggering an error

![image](https://github.com/discourse/discourse/assets/1108771/e51af8c8-a2fa-4a5b-b6db-4d86993b033f)

I believe you need to first save the embeddable host to get an ID and then save the tags, all inside a transaction. Otherwise, you'll probably have to use something like `.tags.build` when creating a new record.